test/setup/MFCompilerTest.php

```diff
@@ -104,7 +113,10 @@ public static function tearDownAfterClass()
 	 * @param string $sThemeDir
 	 * @param ?string $sExpectedReturn
 	 */
-	public function testUseLatestPrecompiledFile(string $sTempTargetDir, string $sPrecompiledFileUri, string $sPostCompilationLatestPrecompiledFile, string $sThemeDir, ?string $sExpectedReturn){
+	public function testUseLatestPrecompiledFile(string $sTempTargetDir, string $sPrecompiledFileUri, string $sPostCompilationLatestPrecompiledFile, string $sThemeDir, ?string $sExpectedReturn, bool $bDisableThemePrecompilationViaConf = false){
+		if ($bDisableThemePrecompilationViaConf){
+			utils::GetConfig()->Set('theme_precompilation_enabled', false);
```

Comment:
```suggestion
			utils::GetConfig()->Set('theme.enable_precompilation', false);
```

Just renaming param. to be up to date with proposal from previous commit